_translator-files/po/zh_CN/1-index.po

```diff
@@ -35,24 +35,39 @@ msgid ""
 "mTOtherPlatforms: 'other platforms'\n"
 "---\n"
 msgstr ""
+"---\n"
+"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
+"lang: \"zh_CN\"\n"
+"permalink: /\n"
+"layout: mainhomepage\n"
+"ldjAppCategory: \"音乐\"\n"
+"ldjSlogan:\"在线播放音乐。与朋友一起。免费的。\"\n"
```

Comment:
There should be a space after the colon.

Reply:
OK

Reply:
Wait for sync